docs/sources/get-started/_index.md

```diff
@@ -56,21 +54,21 @@ As an example, you want to uncover the source of errors in your spans.
 For this, you need to compare the errors in the traces to locate the problem trace.
 Here's how this works.
 
-### Choose a signal type and metric
+### Choose the level of data and a metric
 
-First, you select **Full traces** as the signal type, then choose the **Errors** metric.
-Use **Full traces** to gain insight into the errors in the root of your traces or at the edge of your application.
-If you're interested in any entrypoint to any service or **Database calls** (if you're concerned about databases), use the **Server spans** signal type.
+To identify the troublespot, you want to use raw tracing data instead of just the root span, which is the first span of every trace.
```

Comment:
Should `troublespot` be two words?

Reply:
```suggestion
To identify the trouble spot, you want to use raw tracing data instead of just the root span, which is the first span of every trace.
```